### PR TITLE
Add aviinfrasetting to ako templates

### PR DESCRIPTION
**Pull request by iXinqi**
Add AVIInfraSetting to ako templates so that AVIInfraSetting will be created when addon manager deploys ako

### CODE REVIEW

File: addons/packages/load-balancer-and-ingress-service/1.5.3/bundle/config/upstream/loadbalancerandingressservice/ais.yaml
Comment (XudongLiuHarold) [inline addons/packages/load-balancer-and-ingress-service/1.5.3/bundle/config/upstream/loadbalancerandingressservice/ais.yaml:11]: new line here

Comment (HanFa) [inline addons/packages/load-balancer-and-ingress-service/1.5.3/bundle/config/upstream/loadbalancerandingressservice/ais.yaml:11]: new line here?

**Review by HanFa — DISMISSED** [review]

**Review by XudongLiuHarold — COMMENTED** [review]
<del>We need to hold this until we get everything done and merge them together.</del>

**Review by XudongLiuHarold — CHANGES_REQUESTED** [review]
Need some changes on ako's template

File: addons/packages/load-balancer-and-ingress-service/1.6.1/bundle/config/overlays/overlay-ais.yaml
Comment (iXinqi) [reply addons/packages/load-balancer-and-ingress-service/1.6.1/bundle/config/overlays/overlay-ais.yaml:17]: cidr is not required: https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/blob/release-1.5.2/docs/upgrade/upgrade.md

File: addons/packages/load-balancer-and-ingress-service/1.6.1/bundle/config/overlays/overlay-ais.yaml
Comment (iXinqi) [reply addons/packages/load-balancer-and-ingress-service/1.6.1/bundle/config/overlays/overlay-ais.yaml:13]: You're right about this. We change it to the `service_engine_group_name`.

File: addons/packages/load-balancer-and-ingress-service/1.6.1/bundle/config/upstream/loadbalancerandingressservice/ais.yaml
Comment (iXinqi) [reply addons/packages/load-balancer-and-ingress-service/1.6.1/bundle/config/upstream/loadbalancerandingressservice/ais.yaml:11]: same above

File: addons/packages/load-balancer-and-ingress-service/1.6.1/bundle/config/values.yaml
Comment (iXinqi) [reply addons/packages/load-balancer-and-ingress-service/1.6.1/bundle/config/values.yaml:51]: same above

File: addons/packages/load-balancer-and-ingress-service/1.6.1/bundle/config/upstream/loadbalancerandingressservice/ais.yaml
Comment (XudongLiuHarold) [inline addons/packages/load-balancer-and-ingress-service/1.6.1/bundle/config/upstream/loadbalancerandingressservice/ais.yaml:1]: This should not be in the base template, because when the user doesn't specify any control plane network, you should not create a AIS.

File: addons/packages/load-balancer-and-ingress-service/1.6.1/bundle/config/overlays/overlay-ais.yaml
Comment (XudongLiuHarold) [inline addons/packages/load-balancer-and-ingress-service/1.6.1/bundle/config/overlays/overlay-ais.yaml:4]: You can check if the network name for the control plane is the same as the data network then decide to apply a AIS or not.
You can do something like `ingressclass`here: 
https://github.com/vmware-tanzu/community-edition/blob/main/addons/packages/load-balancer-and-ingress-service/1.6.1/bundle/config/overlays/overlay-statefulset.yaml#L288

https://github.com/vmware-tanzu/community-edition/blob/main/addons/packages/load-balancer-and-ingress-service/1.6.1/bundle/config/overlays/overlay-statefulset.yaml#L305

**Review by XudongLiuHarold — CHANGES_REQUESTED** [review]
Minor changes

**Review by XudongLiuHarold — DISMISSED** [review]
LGTM, thanks!

**Review by XudongLiuHarold — DISMISSED** [review]

**Review by XudongLiuHarold — DISMISSED** [review]
Good catch @HanFa

**Review by XudongLiuHarold — APPROVED** [review]
LGTM

**Pull request merged**
